public/app/features/plugins/admin/pages/Browse.test.tsx

```diff
@@ -94,6 +94,22 @@ describe('Browse list of plugins', () => {
       expect(queryByText('Plugin 2')).not.toBeInTheDocument();
     });
 
+    it('should list installed plugins when filtering by ones with update', async () => {
+      const { queryByText } = renderBrowse('/plugins?filterBy=has-update', [
+        getCatalogPluginMock({ id: 'plugin-1', name: 'Plugin 1', isInstalled: true, hasUpdate: true }),
+        getCatalogPluginMock({ id: 'plugin-2', name: 'Plugin 2', isInstalled: false }),
+        getCatalogPluginMock({ id: 'plugin-3', name: 'Plugin 3', isInstalled: true, hasUpdate: true }),
+        getCatalogPluginMock({ id: 'plugin-4', name: 'Plugin 4', isInstalled: true, isCore: true }),
+      ]);
+
+      await waitFor(() => expect(queryByText('Plugin 1')).toBeInTheDocument());
+      expect(queryByText('Plugin 3')).toBeInTheDocument();
+
+      // Not showing not installed plugins
```

Comment:
**Nit:** `'Plugin 4'` is actually installed, but doesn't have an update, maybe worth updating the comment?